<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newdevicewindow</class>
 <widget class="QDialog" name="newdevicewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new device</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceTypecBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>23</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>HVAC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBLamp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lamp</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Typ urzÄ…dzenia:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviceNameLnEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adres IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviceIPLnEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="thermostatNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa termostatu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="thermostatIpLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adres IP termostatu:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="thermostatNameliEd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="thermostatIpLiEd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newdevicewindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newdevicewindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
